src/shogun/mathematics/linalg/internal/modules/Redux.h

```diff
@@ -36,6 +36,7 @@
 #include <shogun/mathematics/linalg/internal/implementation/Sum.h>
```

Comment:
ehm, please never change file permissions in pull requests.... dont know what happened here